.github/workflows/main.yml

```diff
@@ -8,11 +8,11 @@ jobs:
 
     strategy:
       matrix:
-        php: [8.0, 8.1, 8.2, 8.3]
+        php: [8.0, 8.1, 8.2, 8.3, 8.4]
```

Comment:
please remove support for php 8.0 in this and any future PR where you modify the PHP versions to be tested.